docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -260,6 +219,83 @@ _view=sumologic_slo_output
 | where service="ingestion"
 ```
 
+## Managing your SLOs
+
+Below are some best practices for managing your SLOs. To get to your SLOs list, click **Manage Data** > **Monitoring** > **SLOs** tab.
+
+### Tags and Filters
+
+You can add key/value pair tags to your SLOs to allow you to better organize and filter them. For example, you might find it useful to add tags for `team`, `service`, and `application`.
+
+<!--
+<img src={useBaseUrl('img/observability/FilterByTagKey.png')} alt="FilterByTagKey.png" />
+
+<img src={useBaseUrl('img/observability/FilterByTagValue.png')} alt="FilterByTagValue.png" />
+
+<img src={useBaseUrl('img/observability/DefaultView.png')} alt="DefaultView.png" />
+
+<img src={useBaseUrl('img/observability/ListOfSavedFilters.png')} alt="ListOfSavedFilters.png" />
+
+<img src={useBaseUrl('img/observability/MenuOptionsForAnExistingFilter.png')} alt="MenuOptionsForAnExistingFilter.png" />
+
+<img src={useBaseUrl('img/observability/SaveANewFilter.png')} alt="SaveANewFilter.png" />
+
+<img src={useBaseUrl('img/observability/SaveANewFilterDialogue.png')} alt="SaveANewFilterDialogue.png" />
```

Comment:
@apoorv-garg @tarunk2 , I've added the images and image code here, however, I'm a little unclear about placement. Would you be able to move these image code snippets where you'd like them to be?